recipe/meta.yaml

```diff
@@ -19,17 +19,23 @@ requirements:
     - python {{ python_min }}
     - pip
     - pytest-runner
+    - setuptools >=61
+    - setuptools-scm >=8
```

Comment:
These are presumably required to get the version number right -- it's currently getting reported by `pip check` as 0.0.0.

These are needed according to:
https://github.com/ecmwf/pdbufr/blob/0.14.0/pyproject.toml#L3